tests/xtimer_drift/main.c

```diff
@@ -79,7 +89,10 @@ void *slacker_thread(void *arg)
         struct timer_msg *tmsg = m.content.ptr;
         xtimer_now_timex(&now);
         xtimer_usleep(TEST_MSG_RX_USLEEP);
-
+#if defined(SLACKER_THREAD_PIN)
+        gpio_set(slacker_pin);
+        gpio_clear(slacker_pin);
```

Comment:
Probably a matter of taste, but wouldn't a single toggle be simpler and easier to read on the scope?